### DISCUSSION TITLE
Query Log execution time?

### DISCUSSION THREAD

**Discussion by pk4all** [adonisjs/core#2341] [Help]
Hi,
I have user a query for logging, my insert query taking to much time any help?
My table size is to large approx. 60 GB
```javascript
INSERT INTO `erp_stagging`.`cron_logs` (`execution_time`, `have_heavy_query`, `type`, `created`) VALUES (0, '0', 11, '2021-03-05 05:06:02')
``` 
This query take 3177256 milliseconds.

**Comment by RomainLanz** [comment]
Hey @pk4all! 👋 

What happen when you run the query directly through your DBMS?

**Reply by pk4all** [reply]
Hey @RomainLanz ,
It's take 1004 milliseconds directly through DBMS panel.

**Reply by RomainLanz** [reply]
Could you please share the code you are using to run the query?
